src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -72,7 +72,7 @@ public enum CompressionLevel : int
         {
             NoCompression = 0,
             BestSpeed = 1,
-            BestCompression = 9,    // Refer to "How to choose a compression level" above.
+            BestCompression = 6,    // Refer to "How to choose a compression level" above.
```

Comment:
>Instead of hard-coding our "Optimal" value to be "6" in zlib, I think our "Optimal" value should map to "-1"/"Z_DEFAULT_COMPRESSION" in zlib. 

I agree with that.

The changes will result in a pattern like so:
```
•CompressionLevel.Optimal => ZlibNative.CompressionLevel.DefaultCompression => Z_DEFAULT_COMPRESSION (-1)
•CompressionLevel.Fastest => ZlibNative.CompressionLevel.BestSpeed=> Z_BEST_SPEED (1)
•CompressionLevel.NoCompression => ZlibNative.CompressionLevel.NoCompression=> Z_NO_COMPRESSION (0)
```

Does that look right to you? ZlibNative.CompressionLevel.BestCompression will be removed since we won't be using it anywhere.